test/uk/gov/hmrc/helptosavefrontend/services/HelpToSaveServiceSpec.scala

```diff
@@ -135,12 +135,12 @@ class HelpToSaveServiceSpec extends TestSupport {
         }
 
       "return a successful response" in {
-        List(201, 409).foreach { status ⇒
-          mockCreateAccount(Some(HttpResponse(status)))
-          val result = htsService.createAccount(createAccountRequest)
-          List(result.value.futureValue) should contain oneOf (Right(SubmissionSuccess(false)), Right(SubmissionSuccess(true)))
+        List[(Int, Option[JsValue])]((201, Some(Json.parse("""{"accountNumber" : "1234567890123"}"""))), (409, None)).foreach {
```

Comment:
we are trying to accomodate two scenarios in a single test, why cant we keep separate tests for 201 and 409 ?

Reply:
I thought the original test accommodates both?

Reply:
original test was just dealing with codes, now we've made it complex.. we can separate them into two tests

Reply:
I think its fine how it is, or are you meaning its hard to read?

Reply:
its not a single scenario - its two scenarios,  testing 201 and 409 behaviour
and its hard to read. 
the original one was actually the hacky way of writing that test but it was ok because no returned json and easier to read but now its doing too much , ((otherwise we can accommodate 300 and 400s too in the same test))

Reply:
Ive split this test into two